ethcore/engines/authority-round/src/lib.rs

```diff
@@ -63,8 +63,11 @@ use common_types::{
 	snapshot::Snapshotting,
 };
 use unexpected::{Mismatch, OutOfBounds};
-
 use validator_set::{ValidatorSet, SimpleList, new_validator_set};
+use ethabi::FunctionOutputDecoder;
+use ethabi_contract::use_contract;
+
+use_contract!(block_gas_limit, "../../res/contracts/block_gas_limit.json");
```

Comment:
I think this should rather go to a separate crate that only contains this contract to avoid polluting the engine.